includes/class-wc-stripe-helper.php

```diff
@@ -752,4 +752,85 @@ public static function add_payment_intent_to_order( $payment_intent_id, $order )
 		$order->update_meta_data( '_stripe_intent_id', $payment_intent_id );
 		$order->save();
 	}
+
+	/**
+	 * Adds a source or payment method argument to the request array depending on what sort of
+	 * payment method ID is provided. If ID is neither a source or a payment method ID then nothing
+	 * is added.
+	 *
+	 * @param string $payment_method_id  The payment method ID that should be added to the request array.
+	 * @param array $request             The request representing the arguments that will be sent in the request.
+	 *
+	 * @return array  The updated request array.
+	 */
+	public static function add_payment_method_to_request_array( string $payment_method_id, array $request ): array {
```

Comment:
Another place that could use this helper function:

https://github.com/woocommerce/woocommerce-gateway-stripe/blob/4cee8d98bb9656a244f90901136f79332e306516/includes/abstracts/abstract-wc-stripe-payment-gateway.php#L1695-L1696